### PR TITLE
Formatted hackathon report for gdoc

### PR DESCRIPTION
**Pull request by stefanches7**
After the approval I would upload the materials to the generator http://preview.biohackrxiv.org/

@joshmoore @nolden awaiting your review and OK.

### CODE REVIEW

**Comment by stefanches7** [comment]
@pschader

**Comment by nolden** [comment]
I was happy with the most recent gdoc version, so we can proceed from my side.

**Comment by stefanches7** [comment]
@joshmoore any changes necessary from your side?

**Comment by joshmoore** [comment]
Apologies @stefanches7 and thanks for the ping. In my challenge s, I haven't even had a chance to read the latest. I will ASAP!

File: paper/paper.md [hunk 1]
```diff
@@ -1,158 +1,74 @@
...
 tags:
-  - cheminformatics
-  - PubChem
-  - unknown chemical substances
+  - Bioimaging
+  - publishing
+  - FAIR
 authors:
-  - name: First Author
+  - name: Stefan Dvoretskii
```
Comment (joshmoore) [inline paper/paper.md:9]: Definitely worth your getting an ORCID! :smile:

File: paper/paper.md [hunk 2]
```diff
@@ -1,158 +1,74 @@
...
 ---
 
 
-# Introduction
+# **Introduction**
 
-As part of the BioHackathon Europe 2023, we here report...
+During the SWAT4HCLS 2024 hackathon, we formed a group of participants from various research initiatives and consortia in Bioimaging and Medical Imaging, like the German Biobanking e.V, the Helmholtz Metadata Collaboration (HMC) Hub Health, the Division of Medical Image Computing at the German Cancer Research Center (DKFZ) and the EU Horizon project CCE\_DART (Data Rich Clinical Trials).
 
+With 2024 being the 10-year anniversary of FAIR data, our discussion quickly came to revolve around the state of FAIR access to Bioimaging data, and ideas how to improve it. Within the research bioimaging community, the Open Microscopy Environment (OME) Consortium has been working to standardize access to large-scale imaging data. The OME consortium is a long-term effort which has produced rich software and standards (\>=2000). Imaging Data Resource (IDR) has appeared as an annotation rich imaging portal in 2016\. As of today, it provides access to 129 Studies with more than 400TB data in more that 14 million images, continuously growing. (\[@usesMethodIn:Williams2017\]).
```
Comment (joshmoore) [inline paper/paper.md]: "the 10-year anniversary" yeah?

Comment (joshmoore) [reply paper/paper.md]: `(\>=2000)` I'm not sure what this refers to.

File: paper/paper.md [hunk 3]
```diff
@@ -1,158 +1,74 @@
...
 
+The FAIRness of data in IDR, like most other research data repositories, could be further improved. There are other resources that we would like to connect to, including those outside of the bioimaging domain. RDF is the best mechanism for that connection (\[@citesAsPotentialSolution:moore\_2024\_10687659\]). But a catalog is needed to make the produced RDF triples discoverable.
 
-# Bioimaging dataset aggregation projects
+To facilitate multi-centric research on medical imaging data the Kaapana framework (\[@usesMethodIn:scherer2020joint\]) was created. It provides an open-source framework based on cloud native technologies enabling the large-scale analysis of medical imaging data as well as sharing analysis methods across installations. Primary data as well as analysis results can be stored in the platform and metadata for example from DICOM images is extracted to support cross-center definition and curation of imaging patient cohorts.  Multi center data discovery, like in the DART project of Cancer Core Europe (CCE) also requires approaches for managing and sharing catalogs of metadata. .
```
Comment (joshmoore) [inline paper/paper.md]: At the beginning of this paragraph, it might make sense to describe the history of your group(s).

File: paper/paper.md [hunk 4]
```diff
@@ -1,158 +1,74 @@
...
+To classify the real-world problems with imaging datasets, there was an idea to work on a new “tier” system for publicly available datasets. This tier system would grade the datasets on a scale of their compliance with best data practices. It would complement  existing assessments like the FAIR principles or  the 5-star Open Data system (\[@discusses:Berners-Lee\_2009\]). While the existing tools assess the principal readiness of the data to be FAIR and interlinked, mainly from a technical perspective, the proposed tier system would focus on tangible amount of manual labor that is required to obtain necessary metadata and download public imaging datasets, e.g. finding the right contact person, filling out forms, machine access to metadata or the actual data etc.
```
Comment (joshmoore) [inline paper/paper.md]: Worth cross-linking on of your posters here?

File: paper/paper.md [hunk 5]
```diff
@@ -1,158 +1,140 @@
...
+research initiatives and consortia in bioimaging and medical imaging, including
+German BioImaging e.V, the Helmholtz Metadata Collaboration (HMC) Hub Health,
+the Division of Medical Image Computing at the German Cancer Research Center
+(DKFZ) and the EU Horizon project CCE\_DART (Data Rich Clinical Trials), came
+together.
+
+With 2024 being the 10-year anniversary of FAIR data, our discussion quickly
+came to revolve around the state of FAIR access to bioimaging data, and ideas
+how to improve it. Within the research bioimaging community, the Open
+Microscopy Environment (OME) Consortium has been working to standardize access
```
Comment (joshmoore) [inline paper/paper.md]: should we include links to OME, etc.?

**Review by joshmoore — COMMENTED** [review]
Pushed an initial edit and added some questions for you @stefanches7.

(Finally!, I know. :smile:)

File: paper/paper.md (hunk 1 above)
Comment (stefanches7) [reply paper/paper.md:9]: My bad, I do have one! Please don't tell HMC on this incident :D

File: paper/paper.md (hunk 2 above)
Comment (stefanches7) [reply paper/paper.md]: Thanks for noticing this. I am also not sure what it refers too, I would just remove `(\>=2000)` for now @nolden

File: paper/paper.md (hunk 3 above)
Comment (stefanches7) [reply paper/paper.md]: Thanks! I will find the brief history of MIC and HMC Hub Health and insert them here.

File: paper/paper.md (hunk 4 above)
Comment (stefanches7) [reply paper/paper.md]: Great idea, since they are also on Zenodo 👍

File: paper/paper.md (hunk 5 above)
Comment (stefanches7) [reply paper/paper.md]: Yes, what exactly should I cite? The candidates I have in mind are the 2003 "vision" paper and the 2012 "original" OMERO paper: https://www.openmicroscopy.org/citing-ome/. I incline more towards the 2003 vision.

**Comment by stefanches7** [comment]
@joshmoore asking for a merge, @nolden I hope the history of HMC and MIC are described ok

File: paper/paper.md (hunk 1 above)
Comment (joshmoore) [reply paper/paper.md:9]: No one saw a thing!

File: paper/paper.md (hunk 5 above)
Comment (joshmoore) [reply paper/paper.md]: Definitely ok to use the original, though I meant even more just a footnote to the webpage (i.e. for present usage). Either way.

File: paper/paper.md [hunk 6]
```diff
@@ -7,6 +7,7 @@ tags:
   - FAIR
 authors:
   - name: Stefan Dvoretskii
+    orcid: 0000-0001-7769-0167
```
Comment (joshmoore) [inline paper/paper.md:10]: 🤩

**Comment by joshmoore** [comment]
WFM! Merging and we can always do another round if needed.

**Pull request merged**